sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap.js

```diff
@@ -73,10 +73,9 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         });
     }
 
-    function runOneIteration(useTexSubImage2D, flipY, premultiplyAlpha, bindingTarget, program)
+    function runOneIteration(useTexSubImage2D, bindingTarget, program)
```

Comment:
You can also get rid of useTexSubImage2D as it is always true and this arg is not used in the function at all.